devenv/main.py

```diff
@@ -59,6 +64,10 @@ def devenv(argv: Sequence[str], config_path: str) -> ExitCode:
         # Argparse stuff
         subparser.add_parser(info.command, help=info.help)
 
+    if len(argv) == 1:
+        parser.print_help()
+        return 0
+
```

Comment:
Doesn't argparse print out help by default? Is this help message any different?

Reply:
No, currently `devenv` alone gives

```
usage: devenv [-h] command ...
devenv: error: the following arguments are required: command
```

Reply:
Could we instead utilize the description argument when we construct ArgParse? That way we can utilize argparse's built-in behavior

Reply:
I don't understand what you mean? Also if this can be further improved upon i'd like to followup later.